packages/ui-options/src/Options/Item/index.tsx

```diff
@@ -112,6 +114,9 @@ class Item extends Component<OptionsItemProps> {
     const ElementType = getElementType(Item, this.props, () => as!)
     const passthroughProps = omitProps(this.props, Item.allowedProps)
 
+    const childrenContent = callRenderProp(children)
```

Comment:
This was actually a bugfix, because the children prop allows a function but we just passed the function as children and it broke. It probably never came up as an issue since no one was using functions for the children prop.